examples/speech_recognition/datasets/asr_prep_json.py

```diff
@@ -44,19 +56,17 @@ def main():
                         help="aggregated input labels with format <ID LABEL> per line",
                         type=argparse.FileType('r', encoding='UTF-8'))
     parser.add_argument("--spm-model", required=True,
-                        help="sentencepiece model to use for encoding",
-                        type=argparse.FileType('r', encoding='UTF-8'))
```

Comment:
nit: why not leave `type=argparse.FileType` and get check if file exists for free? `sp.Load(sp_path)` probably does check if file exists but we can get it earlier with argparse

Reply:
TextIOWrapper isn't serializable, so this can't be used.